core_contracts/governance/governance.py

```diff
@@ -90,34 +173,53 @@ def cancelVote(self, name: str) -> None:
         proposal.status.set(ProposalStatus.STATUS[ProposalStatus.CANCELLED])
 
     @external
-    @only_owner
-    def defineVote(self, name: str, description: str, quorum: int, vote_start: int, duration: int, snapshot: int, actions: str,
-                   majority: int = MAJORITY) -> None:
+    def defineVote(self, name: str, description: str, vote_start: int, duration: int, 
+                   snapshot: int, actions: str) -> None:
         """
-        Names a new vote, defines quorum, and actions.
+        Define a new vote and which actions are to be executed if it is successful.
+
+        Parameters:
+        name         -   Name of the vote.
+        description  -   Description of the vote.
+        vote_start   -   Day to start the vote.
+        duration     -   Number of days the vote will be active.
+        snapshot     -   Which day to use for the baln stake snapshot.
+        actions      -   Json string on the form:  {'<action_1>': {<kwargs for action_1>},
```

Comment:
Thanks for improving the docstring here.

Reply:
No problem :)